### ISSUE TITLE
Drag/drop of files from desktop into Maqetta

### ISSUE THREAD

**Issue by jhpedemonte** [maqetta/maqetta#1081]
Allow users to drag/drop files from their desktop into the Files view of Maqetta.

The correct way to do this is with the HTML5 **[File API](https://developer.mozilla.org/en/Using_files_from_web_applications)**.  Unfortunately, it's not supported by all browsers.

@JonFerraiolo

**Comment by jhpedemonte** [comment]
For browsers that don't support the File API yet, **[dropfile.js](https://github.com/MrSwitch/dropfile)** acts as a shim/polyfill to provide the missing functionality, using **Silverlight**.

**Comment by JonFerraiolo** [comment]
I know you aren't recommending dropfile.js, just pointing it out, but I'll just note that we wouldn't want to put in anything that requires Silverlight. Flash is pre-installed on desktop machines, so leveraging Flash without the user knowing we are leveraging Flash might be OK in particular cases (e.g., getting SVG to run on old versions of IE leveraging SVGWeb, which uses Flash on old browsers), but Silverlight has a heavyweight, ugly, scary installation and it would look really bad if our "zero install" product told me they needed to install something that MS is downplaying.

**Comment by jhpedemonte** [comment]
Agreed.  And I'm even hesitant to use Flash.  But I saw that project and wanted to make note of it, for whenever we come around to seriously discussing this feature.

**Comment by jhpedemonte** [comment]
-> M11.

Need to investigate using Orion for this (v2.0 will support drag/drop from desktop).
